backend/demo/src/main/java/com/example/demo/service/stringgeneration/DataType.java

```diff
@@ -0,0 +1,7 @@
+package com.example.demo.service.stringgeneration;
+
+public enum DataType {
+    NAME,
+    MIDDLENAME,
+    SURNAME
```

Comment:
👍 

Пока наверное можно оставить так, в дальнейшем можно подумать над указанием внутри типа данных пути, по которому его можно забрать, получится, что мы вынесем конфигурирование из `DataResource` в `DataType` и сможем налету менять пути

(Все как завещал AmigosCode 🤣 )

```suggestion
public enum DataType {
    NAME("backend/demo/src/main/resources/textFile/name.txt"),
    MIDDLENAME("backend/demo/src/main/resources/textFile/middlename.txt"),
    SURNAME("backend/demo/src/main/resources/textFile/surname.txt");
    
    private final String dataResourcePath;

    DataType(String dataResourcePath) {
        this.dataResourcePath = dataResourcePath;
    }

    public String getDataResourcePath() {
        return dataResourcePath;
    }
}
```